### ISSUE TITLE
Can't pick emoji category via keyboard

### ISSUE THREAD

**Issue by arethsu** [JohnXLivingston/peertube-plugin-livechat#719]
**Description of issue**

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 2.1.1 Keyboard (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/keyboard).

Not everyone can use a mouse, other pointing device, or touchscreen. To make things usable for everyone, it's important for interactive elements to be usable via keyboard alone.

The category selection in the emoji picker receives keyboard focus, but the options can't be activated via keyboard.

![Emoji picker category list, screenshot.](https://github.com/user-attachments/assets/8e8606fe-2fc1-4a95-b157-d9a23f16ae78)

In addition, attempting to select a category using the `Enter` key causes an "Uncaught TypeError" in emoji-picker.js, and if the emoji picker is closed after this it's no longer possible to tab through the page at all.

**To Reproduce**

1. Open the emoji picker and tab forward
2. Try to activate one of the emoji categories

**Recommended solution**

By using correct HTML elements for buttons, links, inputs and other controls, keyboard navigation usually works well by itself. If you use non-semantic elements such as `<div>` or `<span>` with `onClick` functions to build your own components, you must ensure that they can be handled with keyboard only. You must be able to focus on, as well as activate and input values ​​into, all interactive components you build yourself.

Try using the component with only the keyboard. It must work.

**Server:**

 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: macOS 15.5
 - Browser: Safari 18.5

**Additional context**

This issue was part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), under assignment of [HowlRound Theatre Commons](https://howlround.com/).
